packages/editor/src/components/page-attributes/parent.js

```diff
@@ -217,6 +217,13 @@ function PostParentToggle( { isOpen, onClick } ) {
 }
 
 export function ParentRow() {
+	const { homeUrl } = useSelect( ( select ) => {
+		const { getUnstableBase } = select( coreStore );
+		return {
+			homeUrl: getUnstableBase()?.home,
+		};
+	}, [] );
```

Comment:
Adding the current site host is a nice touch.

The only reservation I'd have is for sites with very long domain names. It could push out the UI. In that case something like `example.com|org` could be safer? What do you reckon?

100% optional.

```suggestion
	const { homeUrl } = useSelect( ( select ) => ( {
		homeUrl: select( coreStore ).getUnstableBase()?.home
	} ), [] );
```

Reply:
Probably not a huge risk:


<img width="567" alt="Screenshot 2024-07-05 at 10 14 46 AM" src="https://github.com/WordPress/gutenberg/assets/6458278/5763fa0e-3326-4db9-978e-c8ae2e24d1c3">

Reply:
Yeah, I'm not sure what is better and most clear for the end user: accuracy or brevity. 

For wordpress.com sub-domains it can be quite long, the same will be true for the various make.wordpress.org blogs.

| ![Screenshot 2024-07-05 at 10 22 31 AM](https://github.com/WordPress/gutenberg/assets/519727/f5550a56-0fdd-4311-9b9c-324ab3ff56be) | ![Screenshot 2024-07-05 at 10 25 50 AM](https://github.com/WordPress/gutenberg/assets/519727/93aee297-6283-4cd5-974c-64681fcb3f4f) |

Reply:
Maybe just use `example.com|org`? 

Less code, `0.00001` faster (no `useSelect`) and folks can always do a follow up.

Reply:
Cool, I've pushed that. I'll give others the chance to weigh in before merging.